### ISSUE TITLE
Problem reusing environment on Windows

### ISSUE THREAD

**Issue by drdavella** [tox-dev/tox-conda#19]
Let me begin by saying this is a long awaited, wonderful plugin!

My tox env that once installed the entire scipy stack from pip took 5-10+ mins.  With this plugin - 30+ secs!  Thank you so much @drdavella et al.

*Observation*

I ran into an error re-running tox on Windows.  The first time it seems to run alright.  The second or third time I get an unusual `PermissionError` when conda tries to install `vs2015_runtime-*`.

I imagine this can be reproduced running tox with the following dependencies several times.

```
conda_deps=

    ipykernel
    numpy
    scipy
    pandas
    sympy
```

I can alleviate this error temporarily by deleting the tox env, but of course this is not preferable.  Thanks again!

Update:

While watching the tox env during recreation, I observed the vcruntime dll lingers, doesn't get trashed.  I assume this lingering file is blocking conda installation.

_Originally posted by @pylang in https://github.com/tox-dev/tox-conda/issues/10#issuecomment-478185875_

**Comment by drdavella** [comment]
@pylang thanks for the report. I wonder if you have tested this again with the latest versions of `tox` (3.8.0) and `tox-conda` (0.2.0)?

**Comment by pylang** [comment]
@drdavella Seeing the new release, I pip installed `tox-conda` which updated `tox` to the versions you mentioned.

**Comment by rwhitt2049** [comment]
I can confirm that I'm seeing a similar issue. At some point after multiple re-runs, usually the second or third, I get the following error in my log. This only happens when I include Python 3.5 and Python 3.6 environments.

tox-conda == 0.2.1
tox == 3.16.1
conda==4.8.3
Windows10

```
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
ERROR conda.core.link:_execute(700): An error occurred while installing package 'defaults::vs2015_runtime-14.16.27012-hf0eaf9b_3'.
Rolling back transaction: ...working... done

[Errno 13] Permission denied: 'C:\\python\\log_to_json\\.tox\\py35\\vcruntime140.dll'
```

I usually have a few environments that get created with no issue - usually python 3.7 or newer, but not sure if that's just a coincidence as they are also the last environments to get created. Once I get these errors, I get a new error on the next tox run. I'm assuming it's trying to delete the folder but can't as it's looking for a virtualenv style environment instead of a conda created environment.

```
ERROR: cowardly refusing to delete `envdir` (it does not look like a virtualenv): C:\python\log_to_json\.tox\py35
```

For reference, here's what one of the failed environment folders looks like:
![image](https://user-images.githubusercontent.com/4194594/86938371-fb19f100-c105-11ea-9c32-ca1fee102cbd.png)

If I delete the .tox folder, the next run will work fine.

**Comment by rwhitt2049** [comment]
I can confirm that this is fix in version 0.4. Thanks! I've been fighting that for months now.
